pkg/fission-cli/cmd/mqtrigger/utils.go

```diff
@@ -0,0 +1,18 @@
+package mqtrigger
+
+import "strings"
+
+// UpdateMapFromStringSlice parses key, val from "key=val" string array and updates passed map
+func UpdateMapFromStringSlice(dataMap *map[string]string, params []string) bool {
```

Comment:
Can we move this to Package util in fission-cli instead?

Reply:
Done